config.lua

```diff
@@ -98,7 +98,5 @@ end
 -- usage of stations to players in the same fraction on PvP servers
 -- params: player_name, owner_name, network_name, station_name_start, station_name_target
 travelnet.allow_travel = function()
-	return true
+	return minetest.settings:get_bool("travelnet.allow_travel", true)
```

Comment:
Is this setting useful?

Reply:
cant think of a server ive ever seen run in this config, however i dont see a reason not to expose it to the settings api

Reply:
allows enabling / disabling travel runtime without additional mods / code / overrides, not sure if that's really useful but anyway...

Reply:
seems there is neither a good reason to keep it, or remove it, and it shouldnt? hurt anything, so just going to leave it the way it is